test/support/page_objects/common.js

```diff
@@ -304,10 +304,10 @@ export default class Common {
   }
 
   findTestSubject(selector) {
-    this.debug('in findTestSubject: ' + testSubjSelector(selector));
+    this.debug(`in findTestSubject: ${selector}`);
```

Comment:
Changing the way this works is out of the scope of this pull request. Please put it back the way it was. I'm fine to chat about this, but changing it as a part of this pr isn't the way to do it.